oak_idl_tests/test_schema_services.rs.txt

```diff
@@ -56,8 +56,8 @@ impl <S: TestService> oak_idl::Handler for TestServiceServer<S> {
 }
 
 pub trait TestService: Sized {
-    fn lookup_data(&self, request: &LookupDataRequest) -> oak_idl::utils::Message<LookupDataResponse>;
-    fn log(&self, request: &LogRequest) -> oak_idl::utils::Message<LogResponse>;
+    fn lookup_data(&self, request: &LookupDataRequest) -> Result<oak_idl::utils::Message<LookupDataResponse>, oak_idl::MethodError>;
```

Comment:
How is the error going to be serialized at the framing level? Will it be an additional numeric field in the header of the response?

Reply:
The current framing doc includes a status field, which includes a code for internal errors:
https://docs.google.com/document/d/1Q4u5hYzOFF74_RfwZzpRWgSSwJwfQ-GXRvNllgCXOO8/edit